src/dmd/arraytypes.h

```diff
@@ -67,4 +67,6 @@ typedef Array<class GotoStatement *> GotoStatements;
 
 typedef Array<class TemplateInstance *> TemplateInstances;
 
+typedef Array<class Ensure> Ensures;
```

Comment:
Should this not be `struct Ensure`?

Reply:
It does not make a difference in code semantics, but I guess using `struct` is better style.